doc/command-line.md

```diff
@@ -2011,6 +2032,13 @@ To produce a SQLite manifest, use `sourmash sig manifest ... -F sql`.
 All sourmash commands that work with manifests will accept both
 CSV and SQLite manifest files.
 
+**sourmash v5 breaking change:** In sourmash v5.0 and later,
+`--no-rebuild-manifest` will become default. You can specify `--v4` to
+keep the old behavior, or `--v5` to support the new default behavior,
+or simply specify `--no-rebuild-manifest` to override the default. See
+[Migrating from sourmash v4.x to sourmash v5.x](support.md#migrating-from-sourmash-v4x-to-sourmash-v5x) for more
+details.
+
```

Comment:
Maybe use a github-compatible admonition for these breaking changes? https://mystmd.org/guide/admonitions

```suggestion
> [!WARNING]
> **sourmash v5 breaking change:** In sourmash v5.0 and later,
> `--no-rebuild-manifest` will become default. You can specify `--v4` to
> keep the old behavior, or `--v5` to support the new default behavior,
> or simply specify `--no-rebuild-manifest` to override the default. See
> [Migrating from sourmash v4.x to sourmash v5.x](support.md#migrating-from-sourmash-v4x-to-sourmash-v5x) for more details.

```